UIKitHw/Controllers/BookViewController.swift

```diff
@@ -0,0 +1,213 @@
+    static let typewriterFont = "Typewriter"
+    static let americanTypewriterFont = "AmericanTypewriter"
+    static let americanTypewriterBoldFont = "AmericanTypewriter-Bold"
+    static let helveticaNeueFont = "HelveticaNeue"
+    static let helveticaNeueBoldFont = "HelveticaNeueBold"
+    static let didotFont = "Didot"
+    static let didotBoldFont = "Didot-Bold"
+    static let defaultSizeFont: CGFloat = 20
+    static let defaultIndex = 0
+    static let titleForButtons = "T"
+    static let styleText = ["a", "A"]
+
+  }
+  
+  enum Fonts {
+    static let fonts = [Constants.arialHebrewFont: [UIFont(name: Constants.arialHebrewFont,
+                                                           size: Constants.defaultSizeFont),
+                                                    UIFont(name: Constants.arialHebrewBoldFont,
+                                                           size: Constants.defaultSizeFont)],
+                        Constants.gothicNeoFont: [UIFont(name: Constants.appleSDGothicNeoRegularFont,
+                                                         size: Constants.defaultSizeFont),
+                                                  UIFont(name: Constants.appleSDGothicNeoBoldFont,
+                                                         size: Constants.defaultSizeFont)],
+                        Constants.courierFont: [UIFont(name: Constants.courierFont,
+                                                       size: Constants.defaultSizeFont),
+                                                UIFont(name: Constants.courierBoldFont,
+                                                       size: Constants.defaultSizeFont)],
+                        Constants.typewriterFont: [UIFont(name: Constants.americanTypewriterFont,
+                                                          size: Constants.defaultSizeFont),
+                                                   UIFont(name: Constants.americanTypewriterBoldFont,
+                                                          size: Constants.defaultSizeFont)],
+                        Constants.helveticaNeueFont: [UIFont(name: Constants.helveticaNeueFont,
+                                                             size: Constants.defaultSizeFont),
+                                                      UIFont(name: Constants.helveticaNeueBoldFont,
+                                                             size: Constants.defaultSizeFont)],
+                        Constants.didotFont: [UIFont(name: Constants.didotFont,
+                                                             size: Constants.defaultSizeFont),
+                                                      UIFont(name: Constants.didotBoldFont,
+                                                             size: Constants.defaultSizeFont)]]
+  }
+  
+  // MARK: - Public properties.
+  var currentFont = Constants.arialHebrewFont
+  var selectedSegment = Constants.defaultIndex
+  var bookText = Book()
```

Comment:
```suggestion
 bookTextView.text = Book.staticText
```
если нам от экземпляра нужнo только одно свойство, то тогда лучше сделать его static, раз оно не меняется
 bookTextView.text = bookText.text